ansible/group_vars/all.yml

```diff
@@ -1,14 +1,15 @@
 ---
 ssh_key_name: ""
 target_dns_zone: ""
-target_subdomain: ""
+aws_custom_prefix: ""
 instance_name: ""
 instance_lookup_value: ""
 ec2_install: true
+aws_username: "ec2-user"
 
 #aws_tag_prefix is used to name the various VPC resources so they are only created once per account
 #even if shared with multiple IAM users
-aws_tag_prefix: "test_awsdemo"
+aws_tag_prefix: "{{aws_custom_prefix}}"
```

Comment:
I don't think we want to do this.   It forces each persons deployment to create a new VPC/IGW, etc.
In past we were running with a shared VPC, i.e. we'd create it once for all of the people running in the account.

Since VPCs are often limited, default I think is to have 5 per region, not sure we want to be creating a full replica VPC environment per cluster.